package.json

```diff
@@ -20,5 +21,78 @@
   "license": "MIT",
   "engines": {
     "atom": ">=0.174.0 <2.0.0"
+  },
+  "configSchema": {
+    "executablePath": {
+      "title": "Go Executable Path",
+      "type": "string",
+      "description": "The path to the `go` executable. Find by running `which go`",
+      "default": "go"
+    },
+    "extraOptions": {
+      "type": "string",
+      "default": "-v",
```

Comment:
Is this `-v` _required_ for the linter to work? If so it shouldn't be left up to the user to leave it in there, it should be added later no matter what in the code.

Reply:
I'd  preserved it during the migration from Coffe but I I thought the same thing as you.

Reply:
Ah, so it looks like that enables additional output. The config you added for this works just fine!